backend/src/main/java/com/google/sps/data/Participant.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.data;
+
+/** A user who want to be matched. */
+public final class Participant {
+
+  private final long id;
+  private final String ldap; // Google username
```

Comment:
Like mentioned in PR #9, I'd suggest to rename `ldap` to `username`

Reply:
I'll make this change after all PRs are approved since ldap is used across all files and would be confusing to change in only this PR.

Reply:
sg